media-video/shotcut/Manifest

```diff
@@ -1 +1,2 @@
 DIST shotcut-19.12.31.tar.gz 5475620 BLAKE2B 47fd90ddf40da3c08d3dab31b7170b02aab2f3e6b9d3d7446d53192c04b22b7e053cffe21632f1370d699ff00dca0d057f9a49d29ddeb225f039104abbc8ddee SHA512 714d5b3e184e9d0786bc64635e51fd61570e0404c0ebf0fa5c8a12cb85c9091c75ac61e4963bd400b5722f310ae590b2c2d83aa4cd7acbcb53545f224f0ca766
+DIST shotcut-20.06.28.tar.gz 5797885 BLAKE2B 3f241d6bbe4a89410a2e96ec8bde9446ee88c452ee76f77e4d145f44a37c0d5d116f43e4d0e844b5a112998317f3378c84819bd63c9a02a5aff30338859340b6 SHA512 cd9b96ca467c669138902a94975a5f23baa266b1eb92de0b0446e6f6616a9927aac430c6bd41c4f00de4e5b6d99bfbdf074d339f98ef219a38e3572d8670d78b
```

Comment:
I don't know how you end up with these "no newline" errors, do you perhaps manually edit these files? Please run 
```
rm Manifest
repoman manifest
```
To get it done correctly.